app/services/currency_cleaner.rb

```diff
@@ -0,0 +1,22 @@
+class CurrencyCleaner
+  THOUSANDS_SEPARATOR_REGEX = /,(?=\d{3}\b)/.freeze
+  LEADING_POUND_SIGN_REGEX = /^£/.freeze
+  PURE_NUMERIC_REGEX = /^-?\d+\.?\d+?$/.freeze
+
+  def initialize(value)
+    @original_value = value
+  end
+
+  # will remove all commas if valid i.e. followed by 3 digits
+  # if invalid, the original String is returned
+  def clean
```

Comment:
At the moment, almost all of our services with only one public method follow the pattern of calling that method `call`.
I think it would be good to keep following that pattern.
For example:
`CurrencyCleaner.call` instead of `CurrencyCleaner.clean`
`DataImporter.call` instead of `DataImporter.import`
`EmailSender.call` instead of `EmailSender.send`
etc...